### PR TITLE
issue 397 KMK and DSK wrt card re-issuance

### LINKED ISSUE #397: Key management keys and certificates
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 950-952
 
**Comment (Include rationale for comment)**: "Key management keys and certificates MAY…"
 
**Suggested Change**: "This has dual meaning.  KMK may be generated/certified by CA and injected onto the card.  That is fine.  You could also read this is KMK may not be required when DigSig is required.

Please clarify."
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### PR DESCRIPTION
**Pull request by hferraio**
closes #397

### CODE REVIEW

File: _FIPS201/requirements.md [hunk 1]
```diff
@@ -441,9 +441,8 @@ is reused with the new PIV Card, the digital signature must be recomputed with t
 UUID.
 
 A new PIV authentication certificate and a new card authentication certificate **SHALL** be generated. The
-corresponding certificates **SHALL** be populated with the new FASC-N and card UUID. For cardholders who are
-required to have a digital signature certificate, a new digital signature certificate **SHALL** also be generated.
-Key management keys and certificates **MAY** be imported to the new PIV Card.
+corresponding certificates **SHALL** be populated with the new FASC-N and card UUID. For cardholders with government-issued email account, the digital signature and key management keys and associated certificates are required. A new digital signature key and associated certificate **SHALL** be generated on the new PIV card, while
```
Comment (jimfenton) [inline _FIPS201/requirements.md:444]:
```suggestion
corresponding certificates **SHALL** be populated with the new FASC-N and card UUID. For cardholders with government-issued email accounts, the digital signature and key management keys and associated certificates **SHALL** be populated. A new digital signature key and associated certificate **SHALL** be generated on the new PIV card, while 
```

**Review by jimfenton — CHANGES_REQUESTED** [review]
Minor tweak, thought direct normative language should be used here.

File: _FIPS201/requirements.md (hunk 1 above)
Comment (regenscheid) [reply _FIPS201/requirements.md:444]: Agree with Jim's edits

**Review by regenscheid — CHANGES_REQUESTED** [review]
Agree with Jim's edits.

**Pull request merged**